homeassistant/components/isy994/services.py

```diff
@@ -195,11 +196,39 @@ async def async_system_query_service_handler(service: ServiceCall) -> None:
                     isy.configuration["uuid"],
                 )
                 await isy.query(address)
+
+                # DEPRECATED NOTIFICATION
+                entity_registry = er.async_get(hass)
+                entries_for_this_config = er.async_entries_for_config_entry(
+                    entity_registry, config_entry_id
+                )
+                alternate_target = next(
+                    (
+                        entity.entity_id
+                        for entity in entries_for_this_config
+                        if "query" in entity.entity_id and address in entity.unique_id
+                    ),
+                    "",
```

Comment:
Is the `""` reachable here?

Reply:
Its added as the default for `next()` to guard against a user changing the entity_id of the query button to something without "query" in it. In that case the generator would come up empty and `async_log_deprecated_service_call` is expecting a `str` for that arguement.

Reply:
Maybe look it up by unique id so its always correct?

Reply:
I'm going to pretend I wasn't that stupid.